### PR TITLE
Avoid exception if team info is not present

### PR DESCRIPTION
**Pull request by bklang**

### CODE REVIEW

File: lib/ueberauth/strategy/slack.ex [hunk 1]
```diff
@@ -159,7 +159,7 @@ defmodule Ueberauth.Strategy.Slack do
         auth: conn.private.slack_auth,
         token: conn.private.slack_token,
         user: conn.private.slack_user,
-        team: conn.private.slack_team
+        team: Dict.get(conn.private, :slack_team)
```
Comment (doomspork) [inline lib/ueberauth/strategy/slack.ex]: This should be `Map.get/3`.

**Review by doomspork — CHANGES_REQUESTED** [review]
Thank you @bklang, one small change and we can merge.

File: lib/ueberauth/strategy/slack.ex (hunk 1 above)
Comment (scrogson) [reply lib/ueberauth/strategy/slack.ex]: These could actually all be turned into `conn.private[:slack_*]`.

The square bracket access will just return `nil` for missing keys.

Comment (doomspork) [reply lib/ueberauth/strategy/slack.ex]: @bklang could you incorporate @scrogson's feedback?

**Comment by bklang** [comment]
Sorry, was traveling last week. I'll have a look at this based on the feedback.

File: lib/ueberauth/strategy/slack.ex (hunk 1 above)
Comment (bklang) [reply lib/ueberauth/strategy/slack.ex]: Sorry, was traveling last week. I'll have a look at this based on the feedback.

Comment (doomspork) [reply lib/ueberauth/strategy/slack.ex]: @bklang no worries, I was as well.  Thanks again for contributing your fix.

**Comment by bklang** [comment]
Updated per code review and rebased

**Comment by doomspork** [comment]
Thank you @bklang 👍

**Review by doomspork — APPROVED** [review]

**Pull request merged**